express.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * diva-express
+ * Copyright(c) 2017 Alliander, Koen van Ingen, Timen Olthof
+ * BSD 3-Clause License
+ */
+
+/**
+* Module dependencies.
+* @private
+*/
+
+function requireAttributes(divaSession, attributes) {
```

Comment:
I think this function should take two parameters: `attributes` (the attributes a user has), and `requiredAttributes` (the attributes required). This function will then no longer be dependent on a) a function existing on one of its parameters and b) req.sessionId.

Reply:
We'll fix this once we separate diva-state even more